src/test/scala/chiselTests/RecordSpec.scala

```diff
@@ -146,6 +196,25 @@ class RecordSpec extends ChiselFlatSpec with RecordSpecUtils with Utils {
     e.getMessage should include("contains aliased fields named (bar,foo)")
   }
 
+  they should "be OpaqueType for maps with single unnamed elements" in {
+    val singleElementChirrtl = ChiselStage.emitChirrtl { new SingleElementRecordModule }
+    singleElementChirrtl should include("input in1 : UInt<8>")
+    singleElementChirrtl should include("input in2 : UInt<8>")
+    singleElementChirrtl should include("add(in1, in2)")
+  }
```

Comment:
I forgot, can you also add something checking the `.toTarget` of the opaque type? You can see some examples here: https://github.com/chipsalliance/chisel3/blob/0dd0d06445b97565ad0ee985f3510ffd54aa7971/src/test/scala/chiselTests/ToTargetSpec.scala